.eslintrc.json

```diff
@@ -7,11 +7,13 @@
   },
   "parser": "babel-eslint",
   "extends": ["airbnb", "prettier"],
+  "plugins": ["prettier"],
   "rules": {
     "import/no-extraneous-dependencies": ["error", {"devDependencies": true }],
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/forbid-prop-types": "Off",
     "max-len": ["error", { "code": 80, "ignoreComments": true }],
-    "react/jsx-curly-spacing": ["error", { "when": "never" }]
+    "react/jsx-curly-spacing": ["error", { "when": "never" }],
```

Comment:
you can remove max-len and react/jsx-curly-spacing we don't need it anymore, prettier takes care of it